openapi-bal-service/src/main/java/io/ballerina/openapi/converter/service/OpenAPIParameterMapper.java

```diff
@@ -79,6 +90,17 @@ public void getResourceInputs(Components components, SemanticModel semanticModel
             if (parameterNode.kind() == SyntaxKind.REQUIRED_PARAM) {
                 RequiredParameterNode requiredParameterNode = (RequiredParameterNode) parameterNode;
                 // Handle query parameter
+                if (requiredParameterNode.typeName().kind() == SyntaxKind.QUALIFIED_NAME_REFERENCE) {
+                    QualifiedNameReferenceNode referenceNode =
+                            (QualifiedNameReferenceNode) requiredParameterNode.typeName();
+                    String typeName = (referenceNode).modulePrefix().text() + ":" + (referenceNode).identifier().text();
+                    if (typeName.equals("http:Request")) {
```

Comment:
Instead of using `http:Request` here, shall we introduce a common constant ? I think it will improve the maintainability.